### PR TITLE
Add rna-transcription

### PR DESCRIPTION
**Pull request by BNAndras**
@exercism/ballerina, another #48in24 exercise.

### CODE REVIEW

**Review by DimuthuMadushan — APPROVED** [review]
LGTM!

File: exercises/practice/rna-transcription/rna_transcription.bal [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        } else if c == "T" {
+            complement = "A";
+        } else if c == "A" {
+            complement = "U";
+        }
+
+        results.push(complement);
+    }
+
+    return string:concat(...results);
```
Comment (ThisaruGuruge) [inline exercises/practice/rna-transcription/rna_transcription.bal:18]: Ditto

```suggestion
    map<string> rna = {"C": "G", "G": "C", "T": "A", "A": "U"};
    return from string n in dna
        select rna.get(n);
```

File: exercises/practice/rna-transcription/.meta/reference/rna_transcription.bal [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        } else if c == "T" {
+            complement = "A";
+        } else if c == "A" {
+            complement = "U";
+        }
+
+        results.push(complement);
+    }
+
+    return string:concat(...results);
```
[the remark this answers is not in the document]
Comment (BNAndras) [reply exercises/practice/rna-transcription/.meta/reference/rna_transcription.bal]: To be clear, the example solution isn’t seen by the student unless they come to this repo specifically to look at it. So you may want to submit this as a community solution.

**Review by ThisaruGuruge — APPROVED** [review]

File: exercises/practice/rna-transcription/.meta/reference/rna_transcription.bal (hunk 2 above)
[the remark this answers is not in the document]
Comment (ThisaruGuruge) [reply exercises/practice/rna-transcription/.meta/reference/rna_transcription.bal]: I see. Will add this as a solution as well. Thank you, @BNAndras

**Pull request merged**